### ISSUE TITLE
Rename SwaggerClient to BravadoClient?

### ISSUE THREAD

**Issue by wting** [Yelp/bravado#217]
Since Swagger has been renamed to OpenAPI, any references to Swagger are going to fall farther out of sync. We might as well rename things to `BravadoClient` but this will result in a backwards incompatible change.

**Comment by analogue** [comment]
Good idea. We can create an alias in a backward-compatible minor point release with `SwaggerClient` marked as deprecated.  Then, remove `SwaggerClient` entirely in the next non-backward compatible major release.
